### PR TITLE
PXP-511: [CLI] Develop the connect-instance command

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

screenshot - 2023-05-31T154819 058

Ticket: [PXP-511](https://mindvalley.atlassian.net/browse/PXP-511)

### Added
- [x] add `instances connect` command

[PXP-511]: https://mindvalley.atlassian.net/browse/PXP-511?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/101#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/graphql/mod.rs
Comment (onimsha) [inline src/graphql/mod.rs:407]: I got an error as below 

```
❯ cargo run -- application instances connect green@staging/mv-wukong-ci-mock-green-69b78dfd88-7fj9j
    Finished dev [unoptimized + debuginfo] target(s) in 0.69s
     Running `target/debug/wukong application instances connect 'green@staging/mv-wukong-ci-mock-green-69b78dfd88-7fj9j'`
Checking your permission to connect to the remote instance...✅
⠙ Preparing your remote instance...                                                                                                                                               thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Http(Response { status: 400, version: HTTP/1.1, headers: {"cache-control": "max-age=0, private, must-revalidate", "content-length": "53", "content-type": "application/json; charset=utf-8", "date": "Wed, 31 May 2023 09:31:39 GMT", "server": "Cowboy", "x-request-id": "F2QwguU81p7pNS8AAAQS"}, body: Some([123, 34, 101, 114, 114, 111, 114, 115, 34, 58, 91, 123, 34, 109, 101, 115, 115, 97, 103, 101, 34, 58, 34, 78, 111, 32, 113, 117, 101, 114, 121, 32, 100, 111, 99, 117, 109, 101, 110, 116, 32, 115, 117, 112, 112, 108, 105, 101, 100, 34, 125, 93, 125]) })', /Users/tuanco/code-workspaces/rust/src/github.com/mindvalley/wukong-cli/src/graphql/mod.rs:407:14
```

In production build, how are we handling this error response ?

File: src/commands/application/instances/connect.rs
Comment (onimsha) [inline src/commands/application/instances/connect.rs:108]:
```suggestion
        eprintln!("✅  Your livebook instance is ready! Use the following details to access:\n");
```

File: src/commands/application/instances/connect.rs
Comment (onimsha) [inline src/commands/application/instances/connect.rs:109]:
```suggestion
        eprintln!("URL 🔗 : {}", new_instance.url.unwrap_or_default());
```

File: src/commands/application/instances/connect.rs
Comment (onimsha) [inline src/commands/application/instances/connect.rs:110]:
```suggestion
        eprintln!("Password 🔑 : {}", new_instance.password.unwrap_or_default());
```

File: src/commands/application/instances/connect.rs
Comment (onimsha) [reply src/commands/application/instances/connect.rs:109]: Also, Maybe we format the text `URL` with the green color perhaps ?

File: src/commands/application/instances/connect.rs
Comment (onimsha) [reply src/commands/application/instances/connect.rs:110]: Also, maybe we format the `Password` text with the yellow color perhaps ?

File: src/graphql/mod.rs
Comment (jk-gan) [reply src/graphql/mod.rs:407]: let me check on this

**Review by mfauzaan — APPROVED** [review]
LGTM

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/101#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/101#pullrequestreview-1456986677" title="Approved">mfauzaan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/101#pullrequestreview-1456972599" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
